src/main/java/com/nulabinc/zxcvbn/ZxcvbnBuilder.java

```diff
@@ -0,0 +1,72 @@
+package com.nulabinc.zxcvbn;
+
+import com.nulabinc.zxcvbn.matchers.Dictionary;
+import com.nulabinc.zxcvbn.matchers.Keyboard;
+
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ZxcvbnBuilder {
+
+    private final Map<String, Dictionary> dictionaryMap = new LinkedHashMap<>();
+
+    private final Map<String, Keyboard> keyboardMap = new LinkedHashMap<>();
+
+    public Zxcvbn build() throws IOException {
+        return new Zxcvbn(buildContext());
+    }
+
+    Context buildContext() throws IOException {
+        try {
+            if (this.dictionaryMap.isEmpty() && this.keyboardMap.isEmpty()) {
```

Comment:
It seems that the default dictionary and keyboard is added only when both are EMPTY.
How about checking each the dictionary and keyboard?

Example:

```
if (this.dictionaryMap.isEmpty()) {
    for (Dictionary dictionary : StandardDictionaries.loadAllDictionaries()) {
        this.dictionaryMap.put(dictionary.getName(), dictionary);
    }
}

if (this.keyboardMap.isEmpty()) {
    for (Keyboard keyboard : StandardKeyboards.loadAllKeyboards()) {
        this.keyboardMap.put(keyboard.getName(), keyboard);
    }
}
```

Reply:
@vvatanabe  I designed it that way because I thought it would be difficult to tell when either the dictionary or the keyboard was not set, whether the user had not set it intentionally or had forgotten to set it.
However, let me consider it a bit further to see if there is a way to make it more clearly recognizable to users.